### DISCUSSION TITLE
Multiclass Graph Classification

### DISCUSSION THREAD

**Discussion by maya-poghosyan-old** [stellargraph/stellargraph#1955] [Q&A]
Hi! I am trying to use GCN for multiclass classification. I have three categorical variables in my graph labels. I am following along pretty closely to the tutorial and trying to make some tweaks, however, I am recurrently hitting an error when it comes to creating history with the model.fit function. This is the specific line that keeps giving me an error.

`history = model.fit(train_gen, epochs=epochs, validation_data=test_gen, verbose=2, callbacks=es)`

How can I fix this and use GCN for multi-class graph classification?
